eradiate/scenes/atmosphere/_heterogeneous.py

```diff
@@ -138,60 +143,42 @@ class HeterogeneousAtmosphere(Atmosphere):
 
     profile = documented(
         attr.ib(
-            default=None,
-            converter=RadProfileFactory.convert,
+            default=attr.Factory(AFGL1986RadProfile),
+            converter=attr.converters.optional(RadProfileFactory.convert),
             validator=attr.validators.optional(attr.validators.instance_of(RadProfile)),
         ),
         doc="Radiative property profile used. If set, volume data files will be "
         "created from profile data to initialise the corresponding kernel "
-        "plugin. If ``None``, :class:`.HeterogeneousAtmosphere` will assume "
-        "that volume data files already exist.",
+        "plugin.",
         type=":class:`~eradiate.radprops.rad_profile.RadProfile` or None",
-        default="None",
+        default="AFGL1986RadProfile()",
```

Comment:
This should generate a hyperlink:
```suggestion
        default=":class:`AFGL1986RadProfile() <.AFGL1986RadProfile>`",
```